samples/openapi3/server/petstore/springboot-delegate/src/main/java/org/openapitools/configuration/EnumConverterConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package org.openapitools.configuration;
+
+import org.openapitools.model.EnumClass;
+import org.openapitools.model.OuterEnum;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.convert.converter.Converter;
+
+@Configuration
+public class EnumConverterConfiguration {
+
+    @Bean
+    Converter<String, EnumClass> EnumClassConverter() {
```

Comment:
nit - would expect method name to start with lowercase for Java convention

Reply:
It's changed.